src/commands/update.rs

```diff
@@ -82,13 +83,21 @@ impl UpdateSubcommand {
                         ));
                         false
                     } else {
+                        progress.tick();
```

Comment:
Nitpick: I don't think this is doing what you think it does. `.tick()` updates the spinner to its next character. `.println` will also trigger this rerender behaviour. I think what you want here is `.inc()` which progresses the bar by a given amount.

Either way I'm confused why we are using progressbar here. This shouldn't take long to run. It adds some bulk to the code and I think it would be best to not use it here. Sorry to request that now rather than in the previous review.

Reply:
Yeah, don't worry, it's fixed now!